### PR TITLE
Make Pass be more descriptive (Pass for real passes, Don't or Done otherwise)

### LINKED ISSUE #63: Use Done instead of Pass in appropriate places
Currently, the button that you push to indicate that you (player or corporation) are done performing actions is labeled Pass, but this should be reserved for cases when you are really doing nothing at all; otherwise the button should be called "Done" (or maybe something even more specific like "Lay no track") so as not to imply that you're forfeiting your entire turn.

### PR DESCRIPTION
**Pull request by jenf**
Pass has a specific game meaning in the initial stock round and in stock rounds.

Passing when in the operating round moves to the next step.

Use 'Pass' when this has the specific in game meaning
"Don't" when the user wishes to skip to the next action
"Done" when the user has taking an action in this step (Done Buying Trains)

Resolves #63

### CODE REVIEW

File: lib/engine/round/operating.rb [hunk 1]
```diff
@@ -47,6 +61,7 @@ def initialize(entities, game:, round_num: 1)
         @bankrupt = false
 
         @step = self.class::STEPS.first
+        @lastactionstep = self.class::STEPS.last
```
Comment (tobymao) [inline lib/engine/round/operating.rb:64]: underscores

File: lib/engine/round/operating.rb [hunk 2]
```diff
@@ -65,6 +80,16 @@ def description
         self.class::STEP_DESCRIPTION[@step]
       end
 
+      def pass_description
+        # If the user is continuing an action change the phrasing from Don't to Done
+        if @step == @lastactionstep
```
Comment (tobymao) [inline lib/engine/round/operating.rb]: i don't understand how this works.

isn't last action step always company? you're using the class definition

why would active pass descriptions have multiple things?

Comment (tobymao) [reply lib/engine/round/operating.rb]: ahhhh never mind you set it on line 335

Comment (jenf) [reply lib/engine/round/operating.rb]: step is :track and the like

Comment (tobymao) [reply lib/engine/round/operating.rb]: how about

self.class::ACTIVE_PASS_DESCRIPTION[@step == @last_action_step ? @step : nil] || 
  self.class::INACTIVE_PASS_DESCRIPTION[@step] ||
  'Pass'

Comment (jenf) [reply lib/engine/round/operating.rb]: Ahh, ruby experience shines :)

**Pull request merged**